optaweb-vehicle-routing-frontend/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2019 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.optaweb.vehiclerouting</groupId>
+  <artifactId>optaweb-vehicle-routing-frontend</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>OptaWeb Vehicle Routing Frontend</name>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.github.eirslett</groupId>
+        <artifactId>frontend-maven-plugin</artifactId>
```

Comment:
I noticed some interesting output messages from this plugin:
warnings https://gist.github.com/rsynek/ff9ffc3c2b4a06a89c1bd017c742099a
warnings about deprecations shown as errors https://gist.github.com/rsynek/a529c7d2616e45d27dd4cd5c0f574a4d
information about successfully running all tests presented as an error: https://gist.github.com/rsynek/b1e5baae33b131d20b4d4baa7ba9ebe1

Reply:
Sure, warnings are true npm warnings that can be observed even when running `npm install` from command line. Unrelated to this PR.

Errors are related to this PR because it's a combination of Jest using `stderr` for output and Maven, displaying `stderr` output as `[ERROR]`. Maven's behavior is more or less correct, at least unsurprising. Jest's behavior is debatable. I'd classify it as a bug as multiple people are [running into it](https://github.com/facebook/jest/issues/5064#issuecomment-401451361). Until they fix it, there's nothing we can do about it.

Deprecation warnings displayed as errors - similar to Jest test report. They're valid warnings unrelated to this PR, but displayed as errors thanks to Maven's translation of `stderr`.